source/gent/tier1_hortense.rst

```diff
@@ -216,28 +221,97 @@ This storage space can be addressed with the environment variable $TMPDIR
+rather than your usual VSC home directory (``$VSC_HOME``).
 
-* Please be aware that storage space on $VSC_SCRATCH is limited per user to 3 GB.
-* Instead, it is better to use the dedicated scratch storage space which is reserved for every Tier1 project.
-* The environment variable $VSC_SCRATCH_PROJECTS_BASE points to the basefolder containing all project spaces.
-* Project space is given the same name as your project.
-* You can e.g. use this command to directly switch to your dedicated project space:
+This is done to ensure that Tier-1 Hortense can remain operational, even if there is maintenance being
+performed on the Tier-2 shared storage filesystem of a VSC site (UGent, KUL, VUB, UAntwerpen),
+or in case of problems with the network connection to the other VSC sites.
+
+Although your VSC home directory is usually accessible via ``$VSC_HOME``,
+we strongly recommend to *not* simply create symbolic links to files like your ``.bashrc`` startup script,
+since that would defeat the purpose of this "home-on-scratch" setup.
+
+.. _hortense_project_scratch_dirs:
+
+Project scratch directories
+***************************
+
+* Please be aware that storage space on ``$VSC_SCRATCH`` (personal scratch directory) is limited per user to 3 GB.
+* Instead, it is better to use the dedicated scratch storage space which is reserved for your Tier-1 project.
+* The environment variable ``$VSC_SCRATCH_PROJECTS_BASE`` points to the base folder containing all project directories.
+* Project directories are given the same name as your Tier-1 project (so *without* a prefix like ``gpr_compute_``).
+* To change to your project scratch directory, you can use this command:
 
 .. code:: shell
 
-  cd $VSC_SCRATCH_PROJECTS_BASE/<yourprojectnamehere>
+  cd $VSC_SCRATCH_PROJECTS_BASE/your_project_name
 
+In this command, you should change '``your_project_name``' to the actual name of your project.
 
-Storage quota usage
-*******************
+
+.. _hortense_scratch_storage_quota_usage:
+
+Scratch storage quota usage
+***************************
 
 * You can check personal and project storage quota usage by running the ``my_dodrio_quota`` command.
 * If you want to check storage quota for specific projects, or for projects that are not listed automatically, use the ``-p`` option.
 * For a list of all options, run ``my_dodrio_quota -h``.
 
-*(more information soon)*
+
+.. _hortense_accessing_data_readonly:
+
+Accessing data via ``/readonly``
+********************************
+
+Due to the fairly aggressive page cache purging policy that is used by the `Lustre <https://www.lustre.org>`_
+storage software that is used for the Tier-1 Hortense scratch filesystem, you may need to make some changes
+to how you access data in your job scripts to avoid performance problems.
+
+Whether or not this is required depends on the specific characteristics of your workloads:
+the number of files that are read, how large those files are, how the data in these files is accessed
```

Comment:
it is only relevant for data that is only read more than once. for such data, the perf impact depends on number of files, size, io pattern.